src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -211,4 +210,15 @@ protected function args_to_context( array $arguments, Context $context ) {
 		 */
 		return apply_filters( 'tribe_events_views_v2_list_widget_args_to_context', $alterations, $arguments );
 	}
+
+	public function disable_json_data( $template_vars ) {
```

Comment:
:no_entry_sign: Missing doc comment for function disable_json_data()

<sub>hash: a6cead4572837bbadd1a2b08a81388dd</sub>